util/VarText.h

```diff
@@ -43,6 +43,7 @@ class FO_COMMON_API VarText {
     const std::string&          GetText() const;        //!< Returns text generated after substituting all variables.
     bool                        Validate() const;       //!< Does text generation succeed without any errors occurring?
     const std::string&          GetTemplateString() const   { return m_template_string; }
+    bool                        DoStringtableLookup() const { return m_stringtable_lookup_flag; }
```

Comment:
Should clarify in the name of this function and the member variable what is being looked up in the stringtable... the template string, or the label, or both?